### DISCUSSION TITLE
4.1版本动画播放加速问题

### DISCUSSION THREAD

**Discussion by TJCZJH** [Tencent/libpag#1778] [Q&A]
目前的使用场景是屏幕中同时播放多个相同的动画，两个动画pagFile之间会间隔一段时间进行添加到components，目前遇到一个问题，给pagFile赋值startTime想让动画延迟播放间隔时间是生效的，但是从第二个添加的动画开始会有加速播放的情况，请问是否需要做什么设置来规避这种情况

**Comment by kevingpqi123** [comment]
这类问题建议提供完整的复现 demo 工程解决起来会更快些，避免信息不全

**Comment by kevingpqi123** [comment]
长时间未回复，暂时关闭
